static/src/javascripts/projects/commercial/modules/comscore.spec.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import config from 'lib/config';
+import { init, _ } from './comscore';
+
+const getComscoreScripTags = (): NodeList<HTMLElement> =>
+    document.querySelectorAll('script#comscore');
+
+const isValidComscoreScriptTag = (tag: HTMLElement) =>
```

Comment:
This is only used once, so I'm not certain it's doing much good being separate? It means the reader needs to look in multiple places to understand the test. I'd also point out that this approach means the test fails without telling the developer exactly why (false was not true). Ideally we'd know exactly what was wrong. Was the src wrong? was async false?

This is one reason why it's nice to write tests before implementing a function. It means we get to see them fail before we get to see them succeed and this means that we can practice being the audience for our tests. Do they give us enough feedback to implement the function easily? If they do not, our future colleagues will have the same difficulty maintaining it.

Reply:
👍

Reply:
👍